packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -553,13 +563,49 @@ class DiagnosticsNodeDescription extends StatelessWidget {
   }
 }
 
+/// Label for a property with the default value.
+class DefaultValueLabel extends StatelessWidget {
+  const DefaultValueLabel({
+    super.key,
+    required this.colorScheme,
+    required this.textStyle,
+  });
+
+  final ColorScheme colorScheme;
+  final TextStyle textStyle;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.symmetric(
+        horizontal: denseSpacing,
+      ),
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(7.5),
+        color: colorScheme.inverseSurface,
+      ),
+      child: Text(
+        'default',
+        style: textStyle.merge(
+          TextStyle(
+            fontStyle: FontStyle.normal,
+            color: colorScheme.onInverseSurface,
+            backgroundColor: colorScheme.inverseSurface,
+          ),
+        ),
```

Comment:
use `theme.regularTextStyleWithColor` here instead

Reply:
Done! Modified `regularTextStyleWithColor` to optionally take a `backgroundColor` as well.